const fall = async(req, res) => {
    try{

    }catch(err){

    }
}

export default fall